sdf.md

```diff
@@ -832,6 +832,15 @@ definition specify that the
 interactions offered by a Thing modeled by this sdfThing include the
 interactions modeled by the nested `sdfObject` or `sdfThing`.
 
+## Top-level Affordances and sdfData
+
+Besides their placement within an `sdfObject` or `sdfThing`, affordances
+(i.e., `sdfProperty`, `sdfAction`, and `sdfEvent`) as well as `sdfData` can
+also be placed at the top level of an SDF model.
```

Comment:
```suggestion
also be placed at the top level of an SDF document.
```

SDF model implies sdfObject or sdfThing; now we have the term "document" for "container of definitions"

Reply:
WFM.